TerminalDocs/TOC.yml

```diff
@@ -36,6 +36,8 @@
           href: tutorials/tab-title.md
     - name: Custom Terminal gallery
       items:
+        - name: Custom Terminal Gallery
```

Comment:
I'd recommend changing this to something that's different from the top-level item. Possibly `Custom Terminal guide`?

Reply:
Sounds better!